### PR TITLE
refactor(Presences): Updated Schemas, Fixed ESLint errors & warnings

### PR DESCRIPTION
**Pull request by EncryptedDev**
- Refactor
- Deleted HabboFests, Groovy, Rhythm, flareradio, radio nintendo, the vibe radio, express radio, its pulse, beastfm, tuneradio, madnessfm, boltfm, highlowradio, complexr, curvefm, fiplay, tuga.tv, google play music, RealLifeGTAV, RL.Deals, Ro the Bot, fiplay, plug.dj, discordtrlist.xyz

### CODE REVIEW

**Comment by ImRodry** [comment]
pain

File: package.json [hunk 1]
```diff
@@ -42,5 +42,8 @@
     "source-map-support": "^0.5.19",
     "terser": "^5.9.0",
     "typescript": "^4.4.3"
+  },
+  "dependencies": {
+    "eslint-plugin-eqeqeq-fix": "^1.0.3"
```
Comment (TheGamerzs) [inline package.json:47]: Should this be apart of dev-deps

**Comment by EncryptedDev** [comment]
deleted flareradio, radio nintendo, the vibe radio, express radio, its pulse, beastfm, tuneradio, madnessfm, boltfm, highlowradio, complexr, curvefm, fiplay, tuga.tv, google play music, RealLifeGTAV, RL.Deals, Ro the Bot

**Comment by EncryptedDev** [comment]
delete fiplay

**Comment by EncryptedDev** [comment]
delete plug.dj

**Comment by ImRodry** [comment]
Deleted discordtrlist.xyz and some presences will still fail SV. This is because their links are correct, but the metadata schema must be updated: https://github.com/PreMiD/Schemas/pull/41

**Review by ImRodry — DISMISSED** [review]

File: websites/T/ThreeNow/iframe.ts [hunk 2]
```diff
@@ -4,7 +4,7 @@ iframe.on("UpdateData", () => {
   const video: HTMLVideoElement = document.querySelector(
     "#mw-player_player_html5_api"
   );
-  if (video != undefined) {
+  if (video !== undefined) {
```
[the remark this answers is not in the document]
Comment (Bas950) [reply websites/T/ThreeNow/iframe.ts:7]:
```suggestion
  if (video) {
```

**Review by Slowlife01 — CHANGES_REQUESTED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**